askama_derive/src/generator.rs

```diff
@@ -87,7 +87,7 @@ struct Generator<'a, 'h> {
     /// Set of called macros we are currently in. Used to prevent (indirect) recursions.
     seen_macros: Vec<(&'a Macro<'a>, Option<FileInfo<'a>>)>,
     /// Set of callers to forward into the macro.
-    seen_callers: Vec<&'a Call<'a>>,
+    current_caller: Option<&'a Call<'a>>,
```

Comment:
this is only scoped to the nested callee so this doesn't need to be a stack right. when I do call into another caller i store this onto the stack.